userspace/libsinsp/parsers.cpp

```diff
@@ -2496,7 +2496,7 @@ std::string sinsp_parser::parse_dirfd(sinsp_evt *evt, std::string_view name, int
 	/* This should never happen but just to be sure. */
 	if(name.data() != nullptr)
 	{
-		is_absolute = (name[0] == '/');
+		is_absolute = (name.size() > 0 && name[0] == '/');
```

Comment:
`empty()` has a chance of being more efficient. I'd use that when testing for zero size.
 
```suggestion
		is_absolute = (!name.empty() && name[0] == '/');
```